test/mailers/curator_mailer_test.rb

```diff
@@ -4,30 +4,30 @@ class CuratorMailerTest < ActionMailer::TestCase
 
   test 'released_records' do
```

Comment:
I guess it's not a huge deal, but why not comment out these entire blocks instead of just the stuff in between the do and end?